src/utils/svelte.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Indicates whether the attribute value is a Svelte interpolation.
+ *
+ * ---
+ *
+ * Example interpolation:
+ * ```
+ * a(href="{ cat.id }")
+ * ```
+ *
+ * In this case `val` is `"{ cat.id }"`.
+ *
+ * ---
+ *
+ * @param val Value of tag attribute.
+ * @returns `true` if `val` passes the svelte interpolation check, otherwise `false`.
+ */
+export function isSvelteInterpolation(val: string): boolean {
```

Comment:
This is a copy of `isAngularInterpolation` but with only checking for single curly brackets